seccomp.json

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
Is this the docker default or the projectatomic/docker default.  I would prefer the projectatomi/docker default since we want to support running systemd workloads inside of OCID.